src/go/rpk/pkg/cli/cmd/wasm/generate.go

```diff
@@ -22,15 +22,15 @@ import (
 )
 
 type genFile struct {
-	name		string
-	content		string
-	permission	os.FileMode
+	name       string
+	content    string
+	permission os.FileMode
```

Comment:
Please run `fmt` on this commit.

Reply:
this is the result of `vtools fmt go`

Reply:
after apply I notice a change on that commit, what kind of dark magic is this? :)
Done